packages/graphql/src/translate/translate-read.ts

```diff
@@ -128,16 +128,6 @@ export function translateRead(
         );
     }
 
-    if (!projectionSubqueries.empty && hasOrdering) {
```

Comment:
My understanding of the reason for this conditional block is very limited, the output produced by the TCK tests by removing it, looks good to me, but it may be just of the lack of tests.
Looking at the history seems that this part was touched by the Fulltext improvements and the CypherBuilder migration, does @angrykoala  or @Liam-Doodson  have more clue about it?

Reply:
My understanding of the logic here is also fairly limited. The change I made was to go from a function that added the sorting/pagination using raw cypher, to one that does the same thing using CypherBuilder.

As far as I can tell, there wasn't any logic in the old function that would've avoided this bug though.

Reply:
My main worry is that this may break again the fix for #2267, but as you mention, our lack of tests may have a problem here